manifests/slave.pp

```diff
@@ -170,6 +176,16 @@
     }
   }
 
+  if $slave_args {
+    if is_array($slave_args) {
+      $_combined_slave_args = hiera_array('jenkins::slave::slave_args', $slave_args)
```

Comment:
@jhoblitt fair question. The idea behind this is that there are flags, like `disableClientsUniqueId` and `deleteExistingClients`, that I want to send to the client. I thought of listing these out explicitly but felt this was a bit easier to manage going forward.